packages/react-components/src/components/utils/getParticipantsWhoHaveReadMessage.ts

```diff
@@ -2,6 +2,9 @@
 // Licensed under the MIT license.
 import { ChatMessage, ReadReceiptsBySenderId } from '../..';
 
+/* @conditional-compile-remove(data-loss-prevention) */
+import { BlockedMessage } from '../..';
```

Comment:
Use the specific file instead, especially when it is inside of a package, this will reduce some potential circular dependencies and reduce overall import burdens